nio/client/http_client.py

```diff
@@ -1356,3 +1356,33 @@ def next_response(self, max_events=0):
         self.receive_response(response)
 
         return response
+
+    @connected
+    @logged_in
+    def update_room_topic(
```

Comment:
Let's not add new features to the HTTP client, it'll get deprecated one of these days.

Reply:
Fixed in e0fdf67.